src/script.js

```diff
@@ -86,6 +88,24 @@ $(
       $styleSwitcher.setStyle(styleTitle, $content.contents()[0]);
     });
 
+    var getURLParameter = function(param)
```

Comment:
url query parameters are for server, application state artifacts have no business being exposed like this